docs/New-MarkdownHelp.md

```diff
@@ -5,60 +5,58 @@ schema: 2.0.0
 
 # New-MarkdownHelp
 ## SYNOPSIS
-Convert your existing external help into markdown or generate it from Help object.
+Creates help in markdown format.
+
 ## SYNTAX
 
-### FromModule
+### UNNAMED_PARAMETER_SET_1
```

Comment:
Ok, so that happens when you do Markdown -> MAML -> Markdown convertion.
Which is not the intended workflow like we discussed in https://github.com/PowerShell/platyPS/issues/130#issuecomment-224691734